src/TiciTacaToeyGameEngine.ts

```diff
@@ -11,12 +11,26 @@ import {
   COMPLETED_GAME_STATUS,
   CalculateWinnerInputType,
   CalculateWinnerOutputType,
+  Player,
+  Timer,
 } from "./model";
 import WebSocket = require("ws");
 import uniq from "lodash.uniq";
 
 const EMPTY_POSITION = "-";
+let timePerPlayer = 5000;
```

Comment:
As far as possible, do not use `let` where `const` can be used. Immutability is the source of many bugs.